File: week-02/김용주/01/01.py
import collections
def solution(nums):
    answer = 0
    check = collections.Counter(nums)
    if len(nums)/2 < len(check):
        answer = len(nums)/2
    else:
        answer = len(check)
    return answer
# 지난주의 Counter 함수가 너무 신기해서 써봤는데 
# set이라는 더 좋은 함수가 있더라고요 하하

File: week-12/김용주/01.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers) {
    int answer = 0;
    bool check[11]={0};
    for(int i=0;i<numbers.size();i++)
        check[numbers[i]]=true;
    for(int i=0;i<10;i++)
    {
        if(!check[i])
            answer+=i;
    }
    return answer;
}